src/main/kotlin/com/launchdarkly/intellij/action/ChangeFallthroughAction.kt

```diff
@@ -83,17 +76,24 @@ class ChangeFallthroughAction : AnAction {
             .setItemChosenCallback {
                 ApplicationManager.getApplication().executeOnPooledThread {
                     val settings = LaunchDarklyMergedSettings.getInstance(project)
-                    val patchComment = PatchComment()
-                    val patch = PatchOperation()
                     val currentIdx = parentNode.flag.variations.indexOf(it)
-                    patch.op = "replace"
-                    patch.path = "/environments/" + settings.environment + "/fallthrough/variation"
-                    patch.value = currentIdx
-                    patchComment.patch = listOf(patch)
+                    val flagPatch = PatchOperation().apply {
+                        op = "replace"
+                        path = "/environments/" + settings.environment + "/fallthrough/variation"
+                        value = currentIdx
+                    }
+                    val patchComment = PatchComment().apply {
+                        patch = listOf(flagPatch)
+                    }
                     val ldFlag = LaunchDarklyApiClient.flagInstance(project)
                     try {
                         ldFlag.patchFeatureFlag(settings.project, parentNode.key, patchComment)
                     } catch (e: ApiException) {
+                        val notifier = GeneralNotifier()
```

Comment:
do you think we should catch any exception here? or are we pretty certain ApiException is the main one?

Reply:
same question for all the places we catch apiexception...

Reply:
good point, I'm not totally sure though I'd want us to keep it scoped to just exceptions we generate because another extension/the ide could generate an exception which I don't think we want to handle. So for now I think I'd rather leave it scoped to just `ApiException` unless we get early feedback that there's additional unexpected errors we're not handling.

Reply:
make sense to me